src/go/util/regex_test.go

```diff
@@ -30,7 +31,7 @@ func TestValidateRegexProgramSize(t *testing.T) {
 			desc:        "oversize regex",
 			regex:       "/**",
 			programSize: 1,
-			wantError:   fmt.Errorf("regex program size(5) is larger than the max expected(1): /**"),
+			wantError:   fmt.Errorf("regex program size"),
```

Comment:
why it couldn't do exact match?

Reply:
go/go-brittle-errors. Checking the exact size breaks with this code change or a new go version.